README.md

```diff
@@ -140,6 +144,8 @@ Some metrics are not available with Artifactory OSS license. The exporter return
 
 | Metric | Description | Labels | OSS support |
 | ------ | ----------- | ------ | ------ |
+| artifactory_storage_repo_quota_used_percent | Percentage of quota used |`version`, `revision`, `branch`, `goversion` | ? |
```

Comment:
The list of labels is not correct. You can confirm the list of labels by looking at the metrics we had from the running system before.

Reply:
Additionally, we could use a TODO message or comment specifying that we need to confirm if it works with the OSS version of artifactory.

Reply:
also, we need a way to indicate these 2 metrics are only enabled if the quotas feature is enabled.

Reply:
changed the labels. added the TODO message and created a way to indicate that these two metrics are only enabled if the quota feature is enabled.